organizations/tests/test_api.py

```diff
@@ -667,14 +696,18 @@ def test_add_several_organization_courses(self):
         # Add linkage A->X.
         api.add_organization_course(org_a, course_key_x)
 
+        # Add linkage A->Y, and then remove (actually: deactivate).
+        api.add_organization_course(org_a, course_key_y)
+        api.remove_organization_course(org_a, course_key_y)
+
```

Comment:
Because we no longer re-activate org-courses that are already active, the query count in this test went down, as there were no longer any org-courses to reactivate.

I like the symmetry in the query counts between this test function and the one above it, so I am making one of the org-courses existing-but-inactivate to bring the query count back up to 3.